CHANGELOG.mdx

```diff
@@ -17,6 +17,9 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.1] - 2021-07-20 No significant changes.
```

Comment:
Can we put here that we updated urlib3 to the version that you mentioned in the issue description?

Reply:
done. how does it look?

Reply:
Can we remove "No significant changes."?

Reply:
@tczekajlo done!